From: Yanqing_Liu@Dell.com <Yanqing_Liu@Dell.com>
Date: Wed, 19 May 2010 12:31:36 -0500
Subject: [PATCH] [SCSI] scsi_dh_rdac: Add Dell MD36xxi controller into RDAC device list
X-Git: 2cf75f1c8aa4cab3ace57f9159f1dc85cba69d38
References: FATE#311692
Patch-Mainline: 2.6.34

This patch is to add next generation of Dell iSCSI PowerVault
controller MD36xxi into RDAC device list.

Signed-off-by: Yanqing Liu <Yanqing_Liu@Dell.com>
Signed-off-by: James Bottomley <James.Bottomley@suse.de>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/device_handler/scsi_dh_rdac.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/scsi/device_handler/scsi_dh_rdac.c b/drivers/scsi/device_handler/scsi_dh_rdac.c
index 62511f9..9d736ae 100644
--- a/drivers/scsi/device_handler/scsi_dh_rdac.c
+++ b/drivers/scsi/device_handler/scsi_dh_rdac.c
@@ -767,6 +767,7 @@ static const struct scsi_dh_devlist rdac_dev_list[] = {
 	{"DELL", "MD3000i", 0},
 	{"DELL", "MD32xx", 0},
 	{"DELL", "MD32xxi", 0},
+	{"DELL", "MD36xxi", 0},
 	{"LSI", "INF-01-00", 0},
 	{"ENGENIO", "INF-01-00", 0},
 	{"STK", "FLEXLINE 380", 0},
-- 
1.6.0.2

